include/seqan3/io/stream/debug_stream.hpp

```diff
@@ -295,6 +295,32 @@ inline debug_stream_type & operator<<(debug_stream_type & s, alphabet_t const l)
     return s << to_char(l);
 }
 
+/*!\brief A std::pair can be printed by printing the elements separately.
+ * \tparam pair_t1 Type of first element.
+ * \tparam pair_t2 Type of second element.
+ * \param s The seqan3::debug_stream.
+ * \param p The std::pair.
+ * \relates seqan3::debug_stream_type
+ */
+template <typename pair_t1, typename pair_t2>
+inline debug_stream_type & operator<<(debug_stream_type & s, std::pair<pair_t1, pair_t2> & p)
+//!\cond
+    requires requires (pair_t1 l) { { debug_stream << l }; } && requires (pair_t2 l) { { debug_stream << l }; }
+//!\endcond
+{
+    return s << '<' << p.first << ',' << p.second << '>';
```

Comment:
I think `(,)` would be better delimiters. I would expect that a tuple (also the special case pair) returns `(a,b,c,d)`

Reply:
Maybe, I'll put it in another PR once I'm done. But I need it here for debugging and I might as well already get some feedback on it :D